deploy/kustomize/README.md

```diff
@@ -126,4 +126,4 @@ spec:
 
 #### Deploy
 When customizations are ready, run `./deploy_kustomize.sh`.
-The script will prepare and submit kustimized manifests directories to the cluster. It will also check whenever deployment is complete (HCO CR reports Condition "Available" True), and finish successfully.
\ No newline at end of file
+The script will prepare and submit kustimized manifests directories to the cluster. It will also check whenever deployment is complete (HCO CR reports Condition "Available" True), and finish successfully.
```

Comment:
```suggestion
The script will prepare and submit kustomize manifests to the cluster. It will also check whenever deployment is complete (HCO CR reports Condition "Available" True), and finish successfully.
```

Reply:
done